src/main/java/io/rocketpartners/cloud/action/rest/RestGetAction.java

```diff
@@ -347,7 +360,8 @@ else if (results.size() == limit && (foundRows < 0 || (offest + limit) < foundRo
             }
 
          }
-         expand(req, collection, results.getRows(), null, null, null);
+         if (collection != null)
+            expand(req, collection, results.getRows(), null, null, null);
```

Comment:
prevents expanding if collection is null, as that will cause a null exception. If collection is null, we're likely doing a custom sql from another action, and doesn't need to expand? Since it would know exactly what it wants?